src/styles/index.scss

```diff
@@ -1,3 +1,258 @@
-body {
+@import 'variables';
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: $fw-regular;
+  line-height: 18px;
+  color: $main-text-color;
+}
+
+.container {
+  max-width: 1300px;
+  margin: 0 auto;
+  display: grid;
+
+  box-sizing: border-box;
+}
+
+.header {
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__box {
+    max-width: 1200px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    box-sizing: border-box;
+    padding: 0 50px;
+    height: 60px;
+  }
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.nav {
+  &__menu {
+    display: flex;
+    gap: 20px;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__link {
+    display: inline-block;
+    align-items: center;
+    height: 60px;
+    text-decoration: none;
+    color: $main-text-color;
+    font-size: 12px;
+    text-transform: uppercase;
+    font-weight: $fw-medium;
+    transition-duration: 0.3s;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+
+  &__item {
+    position: relative;
+    line-height: 60px;
+    height: 60px;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.is-active {
+  color: #00acdc;
+
+  &::after {
+    position: absolute;
+    bottom: 0;
+    display: block;
+    content: '';
+    height: 4px;
+    width: 100%;
+    border-radius: 8px;
+    background-color: #00acdc;
+  }
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  transition-duration: 300ms;
+
+  &__title {
+    color: $main-text-color;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 40px;
+    margin-bottom: 4px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    & img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__id {
+    color: $secondary-text-color;
```

Comment:
Ensure that the variable `$secondary-text-color` is defined in the 'variables' file. If not, it may cause a compilation error.